charts/series.go

```diff
@@ -657,3 +662,24 @@ func WithCustomChartOpts(opt opts.CustomChart) SeriesOpts {
 		s.RenderItem = opt.RenderItem
 	}
 }
+
+// WithProgressOpts sets the Progress option.
+func WithProgressOpts(opt opts.Progress) SeriesOpts {
```

Comment:
Thx for add those `WithXXX` helper functions here align current go-echars.
I think remove them are also fine, plz use the `WithSeriesOpts` func instead.
```go
func WithSeriesOpts(opf SingleSeriesOptFunc) SeriesOpts 
```

I plan to remove those `individual` `WithXXX` options in further.
And provide more specific `WithXXXChartsOpts` directly which is more readable and suitable for each chart type.